apps/settings/manifest.webapp

```diff
@@ -44,7 +44,8 @@
         "target": "device"
        },
       "href": "index.html#activity-configure",
-      "disposition": "inline"
+      "disposition": "inline",
+      "returnValue": true
```

Comment:
Sorry @julienw, but I read this message after flag the patch as checkin-needed. Yes, maybe we can change the calls. Do you know where's the right place postResult is called?